doc/intern.rst

```diff
@@ -116,22 +116,24 @@ Look at the file ``lib/system/hti.nim`` for more information.
 Rebuilding the compiler
 ========================
 
-After an initial build via `sh build_all.sh` on posix or `build_all.bat` on windows,
+After an initial build via ``sh build_all.sh`` on posix or ``build_all.bat`` on windows,
```

Comment:
why double backticks? `nim rst2html` and `nim doc` support single backticks and single backticks are simpler (people are more used to markdown)

Reply:
Github preview along with a number of local development tools (eg: the vs code extension that nicely renders it on the side as I work) don't.

Might not want to do it for all docs, but I imagine that a doc like this would often be read in an editor or on GH directly than the final output in a manual. So it definitely doesn't make sense in other docs, but are you sure about this one?

Reply:
=> let's followup here https://github.com/nim-lang/Nim/pull/16341#issuecomment-744103319 to avoid duplicating this discussion